deployment/config.go

```diff
@@ -36,7 +36,7 @@ type Config struct {
 	// Type of the EC2 instance for app.
 	AppInstanceType string `default:"c5.xlarge" validate:"notempty"`
 	// Number of agents, first agent and coordinator will share the same instance.
-	AgentInstanceCount int `default:"2" validate:"range:[1,)"`
+	AgentInstanceCount int `default:"2" validate:"range:[0,)"`
```

Comment:
Is this all that's needed? I remember having to comment certain checks in the past to make this work well.

Reply:
Cool. Would it be possible to fix this case? It could still be useful to show the state of things regardless of agents being created.

https://github.com/mattermost/mattermost-load-test-ng/blob/63a37f5353f808a9b436673a9236eede2f5a3add/deployment/terraform/info.go#L22-L26

Reply:
Yes, indeed that's one thing that I saw. Will fix.